tests/x11/kontact.pm

```diff
@@ -22,9 +22,19 @@ sub run() {
     # To disable it run at first time start
     x11_start_program("echo \"[General]\" >> ~/.kde4/share/config/kmail2rc");
     x11_start_program("echo \"first-start=false\" >> ~/.kde4/share/config/kmail2rc");
-    sleep 2;
+
+    # this is the workaround as same as above but against kontact5
+    # once we got all KF5-based kdepim merged, we can drop the KDE4 workaround
+    x11_start_program("echo \"[General]\" >> ~/.config/kmail2rc");
```

Comment:
As this is a workaround, wouldn't it be better to first check if it's necessary and also include a 'record_soft_failure'?

Are users clearly instructed to do these steps?